### PR TITLE
Feature/56 add webhook to repo when adding a new one

### PR DESCRIPTION
**Pull request by vassilidev**

### CODE REVIEW

File: app/Observers/RepositoryObserver.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            $repository
+                ->webhooks()
+                ->save(
+                    new Webhook([
+                        'title'   => $repository->full_name . ' hook',
+                        'hook_id' => $webhookData->id,
+                    ]),
+                );
+        } catch (Exception $e) {
+            return;
```
Comment (geeksesi) [inline app/Observers/RepositoryObserver.php:31]: shall we do anything else in catch ?

Comment (vassilidev) [reply app/Observers/RepositoryObserver.php:31]: Yes but idk what to do for now, log ? Create without id and cron job to retry? Idk :(

Comment (geeksesi) [reply app/Observers/RepositoryObserver.php:31]: we can flag this kind of repository or PRs and fetch them manually in corn and queue. let's make an issue about it.

Comment (vassilidev) [reply app/Observers/RepositoryObserver.php:32]: https://github.com/geeksesi/code-review-pals/issues/60

**Review by geeksesi — APPROVED** [review]

**Pull request merged**